File: C++/sequentialSearch.cpp
#include <iostream>
#include <random>
#include <cstdio>

int sequentialSearch(int* vetor, int tam, int n) {
    for (int i = 0; i < tam; i++) {
        if (n == vetor[i]) {
            printf("\n%d encontrado na posição %d", n, i);
            return n;
        }
    }
    printf("\nElemento não encontrado no vetor");
    return -1;
}

int main()
{
    
    int tam = 16;
    int* vetor = new int[tam];
    
    for (int i = 0; i < tam; i++) {
        vetor[i] = rand() % 30;
        printf("%d ", vetor[i]);
    }
    
    sequentialSearch(vetor, tam, 23);
    
    delete[] vetor;

    return 0;
}

File: Java/sequentialSearch.java
import java.util.Random;

public class Main
{
    
    public static int sequentialSearch(int vetor[], int n) {
        for (int i = 0; i < vetor.length; i++) {
            if (n == vetor[i]) {
                System.out.printf("\n%d encontrado na posição %d", n, i);
                return i;
            }
        }
        System.out.printf("\nElemento %d não encontrado no vetor", n);
        return -1;
    }
    
	public static void main(String[] args) {
		
        Random rand = new Random();
        
        int[] vetor = new int[16];
        
        for (int i = 0; i < vetor.length; i++) {
            vetor[i] = rand.nextInt(25);
            System.out.printf("%d ", vetor[i]);
        }
        
        sequentialSearch(vetor, 23);
        
	}
}

